src/python/zigopt/pagination/paging.py

```diff
@@ -74,8 +74,91 @@ def _decrement_marker(self, marker):
     marker.symbols[-1].int_value -= 1
     return marker
 
+  def _get_field_values_from_result(self, Field, r):
+    if hasattr(Field, "__call__"):
+      return as_tuple(Field(r))
+    Fields = as_tuple(Field)
+    ret = []
+    for F in Fields:
+      ret.append(getattr(r, F.name))
+    return tuple(ret)
+
+  def _build_marker_from_field_values(self, field_values):
+    marker = PagingMarker()
+    for value in field_values:
+      symbol = marker.symbols.add()
+      if value is None:
+        symbol.null_value = PagingSymbol.NULL_VALUE
+        symbol.null_value  # pylint: disable=pointless-statement
+      elif isinstance(value, dt.datetime):
+        symbol.timestamp_value.FromDatetime(aware_datetime_to_naive_datetime(value))
+      elif is_number(value):
+        if is_integer(value):
+          symbol.int_value = value
+          symbol.int_value  # pylint: disable=pointless-statement
```

Comment:
sidenote: why _do_ we have these?

Reply:
ahh gotcha, interesting.

maybe later, we encapsulate that in a (non-auto-generated) vulture-allowlist somehow, rather than mixed in with code that then causes a different (but more easily solved) issue of pylint complaints.